### ISSUE TITLE
Structure_oM: Add mesh properties to panel object

### ISSUE THREAD

**Issue by rwemay** [BHoM/BHoM#491]
An analytical panel should know how to mesh itself - spacing/mesh type (3 node/4 node), smoothing options etc. Perhaps should be a MeshSettings fragment as we might apply a shell mesh to other objects? @IsakNaslundBh would be great to get this sorted asap, to port across the old workflow code of generate->analyse->pull->update. I can prototype this in CustomData and Robot_Toolkit if necessary?

**Comment by IsakNaslundBh** [comment]
@rwemay think prototyping it up in Robot_Toolkit to start with sounds good to me! think this has come up before with discussion regarding Lusas with @peterjamesnugent , cant remember if you already prototyped something up for it Peter? If so, good to get things aligned to avoid getting things to software-centric.

Also, agree good to get it set up as a fragment, but need to get that system up and running and settled a little bit first :)

**Comment by rwemay** [comment]
Perf. Given complexity definitely needs a good round of prototyping. Will raise issue and link.

**Comment by peterjamesnugent** [comment]
Yeah we have a `MeshSettings1D` and `MeshSettings2D` elements. As @IsakNaslundBh said this would be a good opportunity to standardise the object. Currently, it only contains settings from Lusas, but there's no reason additional settings such as a boolean for smoothing etc. can be added.

https://github.com/BHoM/Lusas_Toolkit/tree/master/Lusas_oM/Structural/Properties

**Comment by IsakNaslundBh** [comment]
To revive this a bit,

Think this could be a good fragment candidate to be implemented after the fragment work in https://github.com/BHoM/BHoM/pull/574 has been done.
